examples/all-clusters-app/all-clusters-common/include/laundry-washer-controls-delegate-impl.h

```diff
@@ -40,11 +40,16 @@ class LaundryWasherControlDelegate : public Delegate
     static LaundryWasherControlDelegate instance;
 
 public:
+    static constexpr uint8_t kDefaultRinseIndex     = 0;
+    static constexpr uint8_t kDefaultSpinSpeedIndex = 0;
```

Comment:
What do these mean?  If they are public API they should be documented.  But are these actually public API?  They seem to be used inside this class only...

Reply:
They are used internally. Will move them to private.